src/App.jsx

```diff
@@ -14,57 +15,60 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [massege, SetMassege] = useState('Jam is selected');
```

Comment:
The function `SetMassege` should be `setMessage` to follow the camelCase naming convention and match the corrected state variable name.